src/org/folio/rest/Okapi.groovy

```diff
@@ -208,6 +208,43 @@ class Okapi extends GeneralParameters {
             throw new AbortException("Error during Elastic Search reindex." + http.buildHttpErrorMessage(res))
         }
     }
+
+    def configureLdpDbSettings(tenant, admin_user, db_host, db_name, db_user, db_password ) {
+        auth.getOkapiToken(tenant, admin_user)
+        String url = okapi_url + "/ldp/config/dbinfo"
+        ArrayList headers = [
+            [name: 'Content-type', value: "application/json"],
+            [name: 'X-Okapi-Tenant', value: tenant.getId()],
+            [name: 'X-Okapi-Token', value: tenant.getAdminUser().getToken() ? tenant.getAdminUser().getToken() : '', maskValue: true]
+        ]
+        logger.info("Starting Configure LDP DB settings")
+        String body = """{"value":"{\\"pass\\":\\"${db_password}\\",\\"user\\":\\"${db_user}\\",\\"url\\":\\"jdbc:postgresql://${db_host}/${db_name}\\"}","tenant":"${tenant.getId()}","key":"dbinfo"}"""
```

Comment:
Maybe better to use template engine here?

Reply:
Reworked with template engine.
@OHaimanov Please check now.